Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:2.7
+
+ADD . /opt/ashlar
+WORKDIR /opt/ashlar
+
+RUN apt-get update
+RUN apt-get -y autoremove
+RUN apt-get install -y libgeos-dev binutils libproj-dev gdal-bin
```

Comment:
What prompted the `autoremove`? I saw it doing a bunch of work, but I wouldn't have thought to do that in the container setup?